### DISCUSSION TITLE
Is there a way to queue up only retries?

### DISCUSSION THREAD

**Discussion by ekahnBP** [spatie/laravel-webhook-server#117] [Q&A]
Basically, we have some financial services (Plaid and Sila) that rely on webhooks in our system. We want the initial webhook handling to be 'sync' (immediate) but any retries because of failure, we would like to queue up as a job.

Is there any way to change the connection type from sync to queue after the initial webhook handling fails?

**Comment by RosiersRobin** [comment]
I'm looking for this too. Not sure if it is possible..
